harmoni_actuators/harmoni_tts/aws_tts_service.py

```diff
@@ -0,0 +1,326 @@
+#!/usr/bin/env python3
```

Comment:
Why is this file here? Should be in nodes or src (seems like this is a copy?)

Reply:
I have no clue about why this is here.